src/table.rs

```diff
@@ -231,9 +231,14 @@ impl Table {
     }
 
     /// Convenience method to set a [ColumnConstraint] for all columns at once.
+    /// Constraints are used to influence the way the columns will be arranged.
+    /// Check out their docs for more information.
     ///
-    /// Simply pass any iterable with ColumnConstraints.\
-    /// If more constraints are passed than there are columns, the superfluous constraints will be ignored.
+    /// **Attention:**
+    /// This function should be called after at least one row has been added to the table.
+    /// Before that, the columns won't initialized.
```

Comment:
Does it have to be a row, or could one set a header, then the constraints?

Reply:
Header would work as well. Good point :)